### DISCUSSION TITLE
The ability to synchronize the list containing people (actors, directors) IMDB and the list Trakt?

### DISCUSSION THREAD

**Discussion by bulletproof2k** [cecobask/imdb-trakt-sync#14] [General]
I was sure that this package synchronizes any item that are in the lists.

**Comment by cecobask** [comment]
Hi @bulletproof2k, on IMDb it's not possible to add people to lists that contain titles.
There are two different types of lists on IMDb - name lists and title lists:
- Name lists contain actors, directors, etc.
- Title lists contain movies, tv shows, etc.

On Trakt you can add people alongside movies and tv shows in the same list.
It is possible to add functionality that enables the sync to happen, but I'm not prioritising it at this moment.

**Reply by bulletproof2k** [reply]
Ok, thx :)
